config/v1/types_dns.go

```diff
@@ -57,7 +57,7 @@ type DNSSpec struct {
 	// infrastructure provider for DNS.
 	// When omitted, this means the user has no opinion and the platform is left
 	// to choose reasonable defaults. These defaults are subject to change over time.
-	// +openshift:enable:FeatureSets=TechPreviewNoUpgrade
+	// +openshift:enable:FeatureSets=CustomNoUpgrade;TechPreviewNoUpgrade
```

Comment:
97802d41772eb7736d894a201ff5b6d3aec42278 doesn't describe the delimiter pivot from `,` to `;`.  Is that a restriction of the kubebuilder syntax?